app/models/category.rb

```diff
@@ -0,0 +1,9 @@
+class Category < ApplicationRecord
+  belongs_to :user
+  has_and_belongs_to_many :trades
+  has_one_attached :icon
```

Comment:
- [ ] As per the project requirements, to create a category, the name and icon must be mandatory, but at the moment I can add a category without an icon. Kindly make an icon a mandatory field.
![image](https://github.com/obed300/budget-app/assets/62849872/2e21c43e-dd51-47ee-a9e9-c5a76e3bdc34)